_posts/2023-10-26-louisvillejefferson-county-metro-government-balance-kentucky/louisvillejefferson-county-metro-government-balance-kentucky.Rmd

```diff
@@ -0,0 +1,135 @@
+---
+title: "Louisville/Jefferson County metro government (balance), Kentucky"
+description: |
+  An analysis of the electoral competitiveness of Louisville Metro Council Elections.
+author: Itsuki Umeyama
+date: 2023-10-26
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+
+## Basic Information
+
+### Demographic
+
+Population[^2]: 386,884
+
+[^2]: United States Census Bureau, *Quick Facts Louisville city, Kentucky; Louisville/Jefferson County metro government (balance), Kentucky*, <https://www.census.gov/quickfacts/fact/table/louisvillecitykentucky,louisvillejeffersoncountymetrogovernmentbalancekentucky/POP010220>
+
+|                           | Percentage |
+|---------------------------|------------|
+| White                     | 67.1%      |
+| Black or African American | 23.8%      |
+| Hispanic or Latino        | 6.5%       |
+| Asian                     | 2.7%       |
+
+![Figure 1: This map shows the proportion of minorities within each precinct in Louisville. Darker shades of blue indicate a higher proportion of minorities, including Black, Hispanic, and Asian populations, living in the precinct compared to the majority, White population. The thinner black lines represent precinct boundaries, while the thicker white lines represent current district boundaries.](images/minority.png){fig-align="center" width="80%"}
```

Comment:
Are these minority voters, or residents? I suggest changing the title to make it clear: "Proportion of Minority" is a bit vague. Maybe say "Proportion of Black, Hispanic, or Asian Residents."